packages/embark-api/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "embark-api",
+  "version": "^4.1.0-beta.0",
```

```suggestion
  "version": "4.1.0-beta.0",
```

Can't have a caret in the version itself, carets are for ranges only, e.g. in `"dependencies"`.